src/profile-logic/committed-ranges.js

```diff
@@ -70,6 +70,5 @@ export function getCommittedRangeLabels(
   const labels = committedRanges.map(range =>
     getFormattedTimeLength(range.end - range.start)
   );
-  labels.unshift('Full Range');
```

Comment:
Does this change the implementation for the non-active tab view? Are we OK with this change?